Editor/Editors/CinemachineVirtualCameraEditor.cs

```diff
@@ -153,16 +163,28 @@ protected void DrawPipelineInInspector()
                 EditorGUI.LabelField(r, label);
                 r = rect; r.width -= labelWidth; r.x += labelWidth;
                 GUI.enabled = !StageIsLocked(stage);
-                int newSelection = EditorGUI.Popup(r, m_stageState[index], sStageData[index].PopupOptions);
+
+                if (targets.Length > 1)
+                    EditorGUI.showMixedValue = !hasSameStageDataTypes[index];
+
                 GUI.enabled = true;
+                EditorGUI.BeginChangeCheck();
+                int newSelection = EditorGUI.Popup(r, m_stageState[index], sStageData[index].PopupOptions);
```

Comment:
GUI.enabled = !StageIsLocked(stage) above is supposed to apply to the popup - you have broken that by setting GUI.enabled = true.

FreeLook, for example, does not allow the user to change the popup value for Body

![image](https://user-images.githubusercontent.com/6424658/88193897-2d4e4700-cc0c-11ea-9534-f8b560abdf29.png)